fix: email and password invalid

When my password is incorrect, the server somehow returns a 500 internal server error, instead of sending me back to the login page with a message on the form saying the email/password combination is invalid.

Although if I look in the server logs, I see the following:
2026-07-14T23:53:25.329Z WARN [Better Auth]: Invalid password
23:53:25 [2026-07-14T23:53:25.332Z] [ERROR] Sign-in failed { message: 'Invalid email or password' }
23:53:25 TypeError: store_get(...).includes is not a function
23:53:25     at $$render_inner (file:///app/build/server/chunks/entries/pages/auth/sign-in/_page.svelte.js-BOJMYtXe.js:54:109)
23:53:25     at file:///app/build/server/chunks/entries/pages/auth/sign-in/_page.svelte.js-BOJMYtXe.js:115:4
23:53:25     at Renderer.child (file:///app/build/server/chunks/chunks/server2.js-13or-APf.js:3552:18)
23:53:25     at Renderer.component (file:///app/build/server/chunks/chunks/server2.js-13or-APf.js:3625:22)
23:53:25     at _page (file:///app/build/server/chunks/entries/pages/auth/sign-in/_page.svelte.js-BOJMYtXe.js:42:13)
23:53:25     at children (file:///app/build/server/chunks/chunks/internal.js-Bfg4GlBy.js:147:13)
23:53:25     at _layout (file:///app/build/server/chunks/entries/pages/auth/_layout.svelte.js-DJ-mCSX8.js:45:2)
23:53:25     at children (file:///app/build/server/chunks/chunks/internal.js-Bfg4GlBy.js:107:9)
23:53:25     at children (file:///app/build/server/chunks/entries/pages/_layout.svelte.js-CimU3KtZ.js:655:5)
23:53:25     at file:///app/build/server/chunks/chunks/tooltip.js-9sYpod0p.js:859:13

Let's fix this error.

--- a/src/routes/auth/sign-in/+page.server.ts
+++ b/src/routes/auth/sign-in/+page.server.ts
@@ -32,11 +32,7 @@ export const actions: Actions = {
 		const form = await superValidate(request, zod4(signInSchema));
 
 		if (!form.valid) {
-			return message(
-				form,
-				{ type: 'error', text: 'Please correct the errors in the form.' },
-				{ status: 400 }
-			);
+			return message(form, 'Please correct the errors in the form.', { status: 400 });
 		}
 
 		try {
@@ -62,7 +58,7 @@ export const actions: Actions = {
 				'An error occurred during sign-in. Please try again.'
 			);
 
-			return message(form, { type: 'error', text: errorMessage }, { status: 400 });
+			return message(form, errorMessage, { status: 400 });
 		}
 	}
 } satisfies Actions;

